### PR TITLE
Update multi-factor-authentication-microsoft-365.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed MFA for individual users to Legacy per person MFA to match other docs.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2516

File: microsoft-365/admin/security-and-compliance/multi-factor-authentication-microsoft-365.md [hunk 1]
```diff
@@ -129,7 +129,7 @@ You can use Azure AD Identity Protection and risk-based Conditional Access polic
 
 For more information, see this [overview of Azure AD Identity Protection](https://docs.microsoft.com/azure/active-directory/identity-protection/overview-identity-protection).
 
-### MFA for an individual user account (not recommended)
+### Legacy Per Person MFA(not recommended)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/multi-factor-authentication-microsoft-365.md]:
```suggestion
### Legacy per person MFA (not recommended)
```

File: microsoft-365/admin/security-and-compliance/multi-factor-authentication-microsoft-365.md [hunk 2]
```diff
@@ -147,7 +147,7 @@ This table shows the results of enabling MFA with security defaults, Conditional
 |:-------|:-----|:-------|:-------|
 | **Security defaults** | Can’t use Conditional Access policies |	Can use Conditional Access policies | Microsoft Authenticator app |
 | **Conditional Access policies** |If any are enabled, you can’t enable security defaults | If all are disabled, you can enable security defaults | User-specified during MFA registration |
-| **Per-user account setting (not recommended)** | Overrides security defaults and Conditional Access policies requiring MFA at each sign in | Overridden by security defaults and Conditional Access policies | User-specified during MFA registration|
+| **Legacy Per Person MFA(not recommended)** | Overrides security defaults and Conditional Access policies requiring MFA at each sign in | Overridden by security defaults and Conditional Access policies | User-specified during MFA registration|
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/security-and-compliance/multi-factor-authentication-microsoft-365.md]:
```suggestion
| **Legacy Per Person MFA (not recommended)** | Overrides security defaults and Conditional Access policies requiring MFA at each sign in | Overridden by security defaults and Conditional Access policies | User-specified during MFA registration|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@Sirkkuw Gentle reminder, please merge this PR. Thanks.

**Pull request merged**